lib/reporter.express.js

```diff
@@ -186,21 +186,20 @@ module.exports = function (reporter, definition) {
   reporter.options.express = definition.options
 
   reporter.options.appPath = reporter.options.appPath || '/'
+  reporter.options.subPath = reporter.options.subPath || '/'

```

Comment:
i don't think it is a good idea to default to `/`, in express using `app.use('/', myapp)` is different than `app.use(myapp)`. just let it undefined if not set